src/App.js

```diff
@@ -0,0 +1,22 @@
+import { Routes, Route, Link } from 'react-router-dom';
+// import axios from 'axios';
+import Greeting from './components/Greeting';
+import './App.css';
+
+function App() {
+  // const response = axios.get('http://127.0.0.1:3000/api/v1/greetings');
+  // const res = response.data;
+  // console.log(res);
+  // fetch('http://127.0.0.1:3000/api/v1/greetings').then((response) => response.json()).then((data) => console.log(data));
```

Comment:
👍 Great job on your project! However, here's a suggestion to enhance the codebase:

- [ ] **Remove Commented-Out Code:** In your codebase, there are multiple instances of commented-out code. It's advisable to avoid leaving commented-out code in the project, as it can clutter the codebase and make it harder to read. Clean code contributes to better maintainability and understanding.

Please go through your project files and remove any commented-out code that is not needed. This practice will help keep your codebase tidy and improve its overall readability.

Keep up the good work! 🚀🧹